zio/zstio/writer.go

```diff
@@ -0,0 +1,35 @@
+package zstio
+
+import (
+	"io"
+
+	"github.com/brimsec/zq/zst"
+)
+
+const (
+	DefaultColumnThresh = 5 * 1024 * 1024
+	DefaultSkewThresh   = 25 * 1024 * 1024
+)
+
+type Writer struct {
+	zst.Writer
+}
+
+type WriterOpts struct {
+	ColumnThresh float64
+	SkewThresh   float64
+}
+
+func MibToBytes(mib float64) int {
+	return int(mib * 1024 * 1024)
+}
+
+func NewWriter(w io.WriteCloser, opts WriterOpts) *Writer {
+	skewthresh := MibToBytes(opts.SkewThresh)
+	colthresh := MibToBytes(opts.ColumnThresh)
+	//XXX should handle error, but zio API doesn't have this yet...
+	// this is just checking bounds on the threshholds so maybe we
+	// do this from above?
+	writer, _ := zst.NewWriter(w, skewthresh, colthresh)
+	return &Writer{*writer}
```

Comment:
Nit: This dereference is a little odd. Is there any reason the embedded zst.Reader field shouldn't just be a *zst.Reader?

Reply:
Same as abovoe

Reply:
WIll return zst.Reader instead of zstio.Reader and get rid of latter